### ISSUE TITLE
I can't find the parameter Cs in code

### ISSUE THREAD

**Issue by LouisDRR** [svg-project/Sparse-VideoGen#76]
Hello author, could you please explain why I cannot find any explicit definition of the spatial head parameter Cs​ in the code?

**Comment by andy-yang-1** [comment]
The parameter is translated to sparsity ratio, see [code](https://github.com/svg-project/Sparse-VideoGen/blob/f0abc563a0b6d3ccc0e718fd1fb892d8ed92c77e/svg/models/wan/utils.py#L63)

**Comment by LouisDRR** [comment]
I still don't understand. The two types of masks generated by the ‘’get_attention_mask‘’ function seem to be used only for online prefilling. The ‘’sparsity‘’parameter is used to generate the block mask for FlexAttention. Suppose we set the parameter ‘’sparsity‘’ to 0.25 and obtain the corresponding ‘’width_frame‘’ via the function ‘’sparsity_to_width’‘. How should this be converted into the spatial head parameter Cs and temporal head parameter Ct for FLOPS calculation? I would be very grateful if you could answer this question.

**Comment by haochengxi** [comment]
https://github.com/svg-project/Sparse-VideoGen/issues/13#issuecomment-2792084039

Please refer to here for more details. Width_frame directly indicates the width of the sparse attention pattern in the spatial head. Temporal Head's sparsity pattern is the same as the spatial head after the Layout Transformation defined in our paper.

**Comment by LouisDRR** [comment]
> [#13 (comment)](https://github.com/svg-project/Sparse-VideoGen/issues/13#issuecomment-2792084039)
> 
> Please refer to here for more details. Width_frame directly indicates the width of the sparse attention pattern in the spatial head. Temporal Head's sparsity pattern is the same as the spatial head after the Layout Transformation defined in our paper.

Thank you！
